importer_utils.py

```diff
@@ -0,0 +1,196 @@
+    if "day" in date_dict:
+        iso_date += "-" + str(date_dict["day"])
+    return iso_date
+
+
+def create_site_instance(language, family):
+    """Create an instance of a Wiki site (convenience function)."""
+    site_key = (language, family)
+    site = site_cache.get(site_key)
+    if not site:
+        site = pywikibot.Site(language, family)
+        site_cache[site_key] = site
+    return site
+
+
+def is_vowel(char):
+    vowels = "auoiyéeöåäáæø"
+    if char.lower() in vowels:
+        return True
+    else:
+        return False
+
+
+def get_last_char(text):
+    """Return the last character of string."""
+    return text[-1]
+
+
+def last_char_is_vowel(text):
+    """Check if last char of string is vowel."""
+    return is_vowel(get_last_char(text))
+
+
+def date_to_dict(datestring, dateformat):
+    """Convert a date to a pwb-friendly dictionary."""
+    date_dict = {}
+    date_obj = datetime.datetime.strptime(datestring, dateformat)
+    date_dict["year"] = date_obj.year
+    if "%m" in dateformat:
+        date_dict["month"] = date_obj.month
+    if "%d" in dateformat:
+        date_dict["day"] = date_obj.day
+    return date_dict
+
+
+def hectares_to_km(hectares):
+    """Convert a value in hectares to km."""
+    one_hectare_in_km = 0.01
+    return one_hectare_in_km * float(hectares)
+
+
+def extract_municipality_name(category_name):
+    """
+    Extract base municipality name from category name.
+
+    Since we can't guess whether an "s" ending the first
+    part of the name belongs to the town name or is
+    a genitive ending (and should be dropped), we have to
+    compare against (English) list of known municipalities:
+
+    Halmstads kommun -> Halmstad
+    Kramfors kommun -> Kramfors
+    Pajala kommun -> Pajala
+
+    Known caveats:
+    * Reserves in Gotland are categorized in "Gotlands län"
+
+    :param category_name: Category of Swedish nature reserves,
+                          like "Naturreservat i Foo kommun"
+    """
+    legit_municipalities = load_json(
+        os.path.join(DATA_DIRECTORY, "municipalities.json"))
+    m = re.search('(\w?)[N|n]aturreservat i (.+?) [kommun|län]', category_name)
+    if m:
+        municipality = m.group(2)
+        municipality_clean = [x["en"].split(" ")[0] for
+                              x in legit_municipalities if
+                              x["sv"] == municipality + " kommun"]
+        if municipality_clean:
+            municipality = municipality_clean[0]
+            if municipality == "Gothenburg":
+                municipality = "Göteborg"
+            return municipality
+
+
+def q_from_wikipedia(language, page_title):
+    """
+    Get the ID of the WD item linked to a wp page.
+    If the page has no item and is in the article
+    namespace, create an item for it.
+    """
+    wp_site = pywikibot.Site(language, "wikipedia")
+    page = pywikibot.Page(wp_site, page_title)
+    summary = "Creating item for {} on {}wp."
```

Comment:
I've always seen it (and spelled it) without a dot. Even https://www.wikidata.org/wiki/Q169514 has "svwp" as an alias, inter alia.